packages/strapi-design-system/src/JSONInput/JSONInput.tsx

```diff
@@ -134,25 +144,3 @@ export const JSONInput = ({ label, value, error, hint, required, onChange, disab
     </Field>
   );
 };
-
-JSONInput.defaultProps = {
-  label: undefined,
-  labelAction: undefined,
-  value: '',
-  error: undefined,
-  hint: undefined,
-  required: false,
-  disabled: false,
-  onChange() {},
```

Comment:
I don't know if it's important but you're not setting these defaults anymore (except onChange)

Reply:
Good point, I guess I was thinking since the rest are all falsy values they weren't necessary? I'll add them back

Reply:
No worries if they're not necessary then, just wanted to make sure this was on purpose